lib/pull.py

```diff
@@ -111,7 +111,9 @@ def download(self):
       # config
       ch.VERBOSE("config path: %s" % self.config_path)
       if (self.config_path is not None):
-         if (os.path.exists(self.config_path) and self.use_cache):
+         if (    os.path.exists(self.config_path)
+             and (ch.file_hash(self.config_path) == self.config_hash)
```

Comment:
This should use the hash in the filename rather than re-computing it. In fact, does this even need to be changed? By the time we call it, won't the config path be up to date with the new hash from the manifest?